src/components/playback/playerSelectionMenu.js

```diff
@@ -1,320 +1,325 @@
+        var getPlayerInfo = playbackManager.getPlayerInfo();
 
-            if (getPlayerInfo) {
-                if (!getPlayerInfo.isLocalPlayer && getPlayerInfo.supportedCommands.indexOf('DisplayContent') !== -1) {
-                    mirrorItem(info, playbackManager.getCurrentPlayer());
-                }
+        if (getPlayerInfo) {
+            if (!getPlayerInfo.isLocalPlayer && getPlayerInfo.supportedCommands.indexOf('DisplayContent') !== -1) {
+                mirrorItem(info, playbackManager.getCurrentPlayer());
             }
         }
     }
+}
 
-    function emptyCallback() {
-        // avoid console logs about uncaught promises
-    }
+function emptyCallback() {
+    // avoid console logs about uncaught promises
+}
 
-    function getTargetSecondaryText(target) {
+function getTargetSecondaryText(target) {
 
-        if (target.user) {
-
-            return target.user.Name;
-        }
+    if (target.user) {
 
-        return null;
+        return target.user.Name;
     }
 
-    function getIcon(target) {
+    return null;
+}
 
-        var deviceType = target.deviceType;
+function getIcon(target) {
 
-        if (!deviceType && target.isLocalPlayer) {
-            if (browser.tv) {
-                deviceType = 'tv';
-            } else if (browser.mobile) {
-                deviceType = 'smartphone';
-            } else {
-                deviceType = 'desktop';
-            }
-        }
+    var deviceType = target.deviceType;
 
-        if (!deviceType) {
+    if (!deviceType && target.isLocalPlayer) {
+        if (browser.tv) {
             deviceType = 'tv';
-        }
-
-        switch (deviceType) {
-
-            case 'smartphone':
-                return 'smartphone';
-            case 'tablet':
-                return 'tablet';
-            case 'tv':
-                return 'tv';
-            case 'cast':
-                return 'cast';
-            case 'desktop':
-                return 'computer';
-            default:
-                return 'tv';
+        } else if (browser.mobile) {
+            deviceType = 'smartphone';
+        } else {
+            deviceType = 'desktop';
         }
     }
 
-    function showPlayerSelection(button) {
+    if (!deviceType) {
+        deviceType = 'tv';
+    }
 
-        var currentPlayerInfo = playbackManager.getPlayerInfo();
+    switch (deviceType) {
+
+        case 'smartphone':
+            return 'smartphone';
+        case 'tablet':
+            return 'tablet';
+        case 'tv':
+            return 'tv';
+        case 'cast':
+            return 'cast';
+        case 'desktop':
+            return 'computer';
+        default:
+            return 'tv';
+    }
+}
 
-        if (currentPlayerInfo) {
-            if (!currentPlayerInfo.isLocalPlayer) {
-                showActivePlayerMenu(currentPlayerInfo);
-                return;
-            }
-        }
+export function showPlayerSelection(button) {
```

Comment:
Taking into account `show: showPlayerSelection` below, you need to rename function to `show`.